server/sonar-server/src/main/java/org/sonar/server/computation/issue/ScmAccountCacheLoader.java

```diff
@@ -19,28 +19,53 @@
  */
 package org.sonar.server.computation.issue;
 
+import com.google.common.base.Function;
+import com.google.common.base.Joiner;
+import com.google.common.collect.Collections2;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.sonar.server.user.index.UserDoc;
 import org.sonar.server.user.index.UserIndex;
 import org.sonar.server.util.cache.CacheLoader;
 
 import java.util.Collection;
+import java.util.List;
 import java.util.Map;
 
 /**
  * Loads the association between a SCM account and a SQ user
  */
-public class ScmAccountCacheLoader implements CacheLoader<String,String> {
+public class ScmAccountCacheLoader implements CacheLoader<String, String> {
 
+  private final Logger log;
   private final UserIndex index;
 
   public ScmAccountCacheLoader(UserIndex index) {
+    this(index, LoggerFactory.getLogger(ScmAccountCacheLoader.class));
+  }
+
+  public ScmAccountCacheLoader(UserIndex index, Logger log) {
```

Comment:
the visibility should be package-protected and add the annotation that it's for testing purpose